creations/weact.md

```diff
@@ -0,0 +1,16 @@
+# Wemos Creations
+
+Community Allocated Creation IDs for [WeAct boards](https://weactstudio.aliexpress.com/store/910567080)
+
+Creator ID : 0x1011_1011
+
+## `0x0032_xxxx` - ESP32 dev boards
+
+## `0x0052_xxxx` - S2 dev boards
+
+## `0x0053_xxxx` - S3 dev boards
+
+## `0x00C3_xxxx` - C3 dev boards
+
+## `0x00C6_xxxx` - C6 dev boards
+*  `0x00C6_0001` [WeAct ESP32C6](https://www.aliexpress.com/item/1005005569520224.html)
```

Comment:
Please do one number for each flash variant. (As I said on the CP PR.)

Reply:
I did, now it's 04 for N4 and 08 for N8.
If there are more C6 boards in the future, we can do 14, 18 etc.

Reply:
For 16, we could do "a", for 32 "b" and etc.